server/config/toml.go

```diff
@@ -47,6 +47,10 @@ halt-time = {{ .BaseConfig.HaltTime }}
 # InterBlockCache enables inter-block caching.
 inter-block-cache = {{ .BaseConfig.InterBlockCache }}
 
+# IndexEvents defines the set of events in the form {eventType}.{attributeKey},
+# which informs Tendermint what to index. If empty, all events will be indexed.
```

Comment:
are they comma separated? would be good to define

Reply:
its an array, so yes, naturally.

Reply:
I understand this but it is a question we get on Tendermint so its better to specify